desktop-environment.el

```diff
@@ -182,6 +192,44 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % of maximum backlight."
+  (let ((backlight-level (desktop-environment-keyboard-backlight-get)))
+    (if (eq backlight-level 0)
+        "0.0"
+      (*
+       (/ (* backlight-level 1.0)
+          (* (desktop-environment-keyboard-backlight-get-max) 1.0))
+       100))))
+
+(defun desktop-environment-keyboard-backlight-get ()
+  "Return a number representing keyboard backlight current level."
+  (dbus-call-method :system
```

Comment:
please add a `require` for dbus line at the beginning of the file

Reply:
Added:
`(require 'dbus)`